### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Filegroup_VoiceOver: Voice over is announcing all the column header information twice which is present on "Rows" "File Stream" "Memory Optimized data" grid.

### ISSUE THREAD

**Issue by HindujaArugula** [microsoft/azuredatastudio#25141]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev1; #DesktopApp; #WCAG1.3.1; #FTP; A11yTCS, #SH_Azure Data Studio_Nov2023_Mac_Dec2023; #SQL Azure Data Studio; #MAC; #VoiceOver;

### Environment Details:
Application Name: Azure Data Studio
Operating System: MAC
Azure data studio 1.47.0
VoiceOver

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Tab till "Database" and press ENTER to expand the tree.
4. Tab till any database and pressF10 to open context menu.
5. Tab till "Properties Preview" and press ENTER.
6. Tab till "FilesGroup" tab and press ENTER.
7. Tab till "Database files" table.
8. You will observe that voice over is announcing all the column header information twice which is present on "Rows" "File Stream" "Memory Optimized data" grid.

### Actual Result:
When navigating to "Database files" table voice over is announcing all the column header information twice which is present on "Rows" "File Stream" "Memory Optimized data" grid.

**Note: Issue seen throughout the application for similar tables.**

### Expected Result:
When navigating to "Database files" table voice over should not announce all the column header information twice which is present on "Rows" "File Stream" "Memory Optimized data" grid.

User Impact:
Screen Reader users will face difficulties if the voiceover is announcing the information of the column headers twice.

### Attachment:
![Voice Over grid](https://github.com/microsoft/azuredatastudio/assets/125462007/c293fe36-9963-4707-967a-6688417f6a95)

**Comment by kisantia** [comment]
@HindujaArugula the same reported behavior with the table header was previously filed and closed as non-blocking after attending the accessibility office hours. Can this issue be closed since the previous issue with the same behavior was approved to be closed?
